city_scrapers/spiders/chi_ssa_4.py

```diff
@@ -0,0 +1,137 @@
+from datetime import datetime
+
+import scrapy
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from dateutil.relativedelta import relativedelta
+
+
+class ChiSsa4Spider(CityScrapersSpider):
+    name = "chi_ssa_4"
+    agency = "Chicago Special Service Area #4 South Western Avenue"
+    timezone = "America/Chicago"
+
+    def start_requests(self):
+        today = datetime.now()
+        for month_delta in range(-6, 3):
+            mo_str = (today + relativedelta(months=month_delta)).strftime("%Y-%m")
+            url = "http://95thstreetba.org/events/category/board-meeting/{}/".format(
+                mo_str
+            )
+            yield scrapy.Request(url=url, method="GET", callback=self.parse)
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        for url in self._get_event_urls(response):
+            yield scrapy.Request(url, callback=self._parse_event, dont_filter=True)
+
+    def _parse_event(self, response):
+        """Parse the event page"""
+        meeting = Meeting(
+            title=self._parse_title(response),
+            description=self._parse_description(response),
+            classification=COMMISSION,
+            start=self._parse_start(response),
+            end=self._parse_end(response),
+            all_day=False,
+            time_notes=self._parse_time_notes(response),
+            location=self._parse_location(response),
+            links=self._parse_links(response),
+            source=response.url,
+        )
+
+        meeting["status"] = self._get_status(meeting)
+        meeting["id"] = self._get_id(meeting)
+        return meeting
+
+    def _get_event_urls(self, response):
+        """
+        Get urls for all meetings on the calendar page
+        """
+        urls = []
+        links = response.css(
+            "table td.tribe-events-thismonth h3 a::attr(href)"
+        ).extract()
+        descriptions = response.css(
+            "table td.tribe-events-thismonth h3 a::text"
+        ).extract()
+
+        for link, description in zip(links, descriptions):
+            if "meeting" in description.lower():
+                urls.append(link)
+        return urls
+
+    def _parse_title(self, response):
+        """Parse or generate meeting title."""
+        return "".join(response.css("div.tribe-events-single h1::text").get())
+
+    def _parse_description(self, response):
+        """Parse or generate meeting description."""
+        selector = "div.tribe-events-single-event-description p::text"
```

Comment:
It looks like this is always the same so it should be fine to ignore and return an empty string instead

Reply:
In this case, do you think it would be more efficient to get rid of the `_parse_description` method entirely and put `description=""` in the `_parse_event` method?

Reply:
That works! That's usually what I end up doing, but it's a preference thing